packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -4657,23 +4657,23 @@ curl -v -X GET \
 
 <ApiLifecycle access="ea" />
 
-A scope consent Grant represents permission for the application to request that Okta scopes be included in OAuth 2.0 Bearer tokens. Without this Grant, token requests that contain Okta scopes are denied.
+A scope consent grant represents an application's permission to request to include a specific Okta scope in OAuth 2.0 Bearer tokens. If the application does not have this grant, token requests that contain this Okta scope are denied.
 
 ### Grant Consent to Scope for Application
 
 <ApiLifecycle access="ea" />
 
 <ApiOperation method="post" url="/api/v1/apps/${applicationId}/grants" />
 
-Grants consent for the application to request OAuth 2.0 Okta scopes
+Grants consent for the application to request an OAuth 2.0 Okta scope.
 
 #### Request Parameters
 
 | Parameter       | Description                                                                                    | Param Type   | DataType   | Required   | Default |
 | :-------------- | :--------------------------------------------------------------------------------------------- | :----------- | :--------- | :--------- | :------ |
 | applicationId   | ID of the application                                                                          | URL          | String     | TRUE       |         |
-| issuer          | The issuer of your Org Authorization Server, your Org URL                                    | Body         | String     | TRUE       |         |
-| scopeId         | The name of the Okta scope for which consent is to be granted                                      | Body         | String     | TRUE       |         |
+| issuer          | The issuer of your Org Authorization Server, your Org URL                                      | Body         | String     | TRUE       |         |
+| scopeId         | The name of the Okta scope for which consent is granted to                                     | Body         | String     | TRUE       |         |
```

Comment:
don't need the word "to" on the end to keep the meaning....

The name of the Okta scope for which consent is granted